_rules/SC4-1-1+SC4-1-2-aria-allowed-attribute.md

```diff
@@ -19,7 +19,7 @@ authors:
 
 ### Applicability
 
-Any attribute that is specified on an HTML or SVG element where the attribute name starts with `aria-`.
+Any [WAI-ARIA state or property](https://www.w3.org/TR/wai-aria-1.1/#state_prop_def) that is specified on an HTML or SVG element.
```

Comment:
What about MathML?

Reply:
We haven't included it in any other rule so far. I suggest we limit to html/svg for now and then take an overall discussion on which rules should consider MathML and other techonologies.

Reply:
That's not the case though; several rules already consider MathML implicitly as also mentioned in https://github.com/auto-wcag/auto-wcag/issues/312.

Reply:
I suggest we merge this one in as is and then fix ALL applicabilities to be consistent when it comes to inclusion/exclusion of HTML, SVG and MathML